cmd/root.go

```diff
@@ -185,6 +146,17 @@ func initConfig() {
 	// config logger
 	logConfig(cfg)
 
+	// ovverride config with fields from the inputConfig
+	if inputConfig.GroupName != "" {
+		cfg.GroupMatch = inputConfig.GroupName
+		log.Infof("Overriding GroupMatch with %s", inputConfig.GroupName)
+	}
+
+	// if inputConfig.UserName != "" {
+	// 	cfg.UserMatch = inputConfig.UserName
+	// 	log.Infof("Overriding UserMatch with %s", inputConfig.UserName)
+	// }
+
```

Comment:
do we want to keep this here?

Reply:
oh I see - if you want to leave this here maybe we can add a comment on how to use it

Reply:
I can add a comment, wanted to include it incase we ever wanted to get it to the user level, will add a note